kivy/tools/packaging/pyinstaller_hooks/pyi_rth_kivy.py

```diff
@@ -5,8 +5,7 @@
 
 os.environ['KIVY_DATA_DIR'] = os.path.join(root, 'data')
 os.environ['KIVY_MODULES_DIR'] = os.path.join(root, 'modules')
-os.environ['GST_PLUGIN_PATH'] = '{};{}'.format(
-    sys._MEIPASS, os.path.join(sys._MEIPASS, 'gst-plugins'))
+os.environ['GST_PLUGIN_PATH'] = os.path.join(sys._MEIPASS, 'gst-plugins')
```

Comment:
Very nice!

Reply:
Why this change? On Windows the plugins are not in a separate directory so the root needs to be on the path. Was there an error on osx that made you change this?

Reply:
Ah, I didn't realize the plugins were stored in the package root in Windows. If I remember correctly, the error I was seeing on macOS was due to the semicolon instead of a colon, and I followed this PR: https://github.com/pyinstaller/pyinstaller/pull/1963

Reply:
No worries. This was fixed in https://github.com/kivy/kivy/pull/6359/.